src/app/app.routing.ts

```diff
@@ -9,51 +9,48 @@ import { PasswordResetComponent } from './pages/password-reset/password-reset.co
 import { EnterResetPasswordComponent } from './pages/enter-reset-password/enter-reset-password.component';
 import { NoAuthGuard } from './guards/no-auth.guard';
 
-
 const routes: Routes = [
   {
-    path: '',
-    redirectTo: 'home',
-    pathMatch: 'full'
+    path: ``,
```

Comment:
Hey @Genza999, what's our motivation for changing to using template literals for all the routes configurations.

Reply:
Hey @CryceTruly , It was an issue I was trying to fix, but I have now reverted back to the previous changes.